addons/account/models/account_move.py

```diff
@@ -3433,6 +3435,12 @@ def _get_move_display_name(self, show_ref=False):
         :return:            A string representing the invoice.
         '''
         self.ensure_one()
+        if self.env.context.get('input_duplicate_warning_display_name'):
```

Comment:
I think the current name of the key is pretty specific and would be weird to reuse elsewhere.
It might be better to name it with something a little bit more generic like `name_as_amount_total`. (or something better)

This way, it wouldn't be weird if the context key is set somewhere else and other people won't be reluctant/afraid to use it.